.github/workflows/ci-operator-release.yaml

```diff
@@ -0,0 +1,43 @@
+name: ci-release-operator
+
+on:
```

Comment:
If KubeArmorOperator will be the main source for installing KubeArmor, I think we'll also need to publish stable images on tag releases? :thinking:

Reply:
we'll handle it later as we need to refactor our existing CI release workflow.